group_project_v2/mixins.py

```diff
@@ -96,7 +96,8 @@ def user_id(self):
         try:
             return int(self.real_user_id(self.anonymous_student_id))
         except Exception as exc:
-            log.exception(exc)
+            if getattr(self.runtime, 'is_author_mode', False):
+                log.exception(exc)
```

Comment:
We want the exact opposite of this, i.e. don't log the error if it's in author mode.